tests/compile/test_full_graph.py

```diff
@@ -20,14 +20,14 @@ def models_list(*, all: bool = True, keywords: Optional[list[str]] = None):
         ("facebook/opt-125m", {}),
         ("nm-testing/tinyllama-oneshot-w8w8-test-static-shape-change", {
             "dtype": torch.float16,
-            "quantization": "compressed-tensors"
+            "quantization": "compressed_tensors"
         }),
         ("neuralmagic/Llama-3.2-1B-Instruct-FP8-dynamic", {
             "dtype": torch.float16,
-            "quantization": "compressed-tensors"
+            "quantization": "compressed_tensors"
         }),
         ("neuralmagic/Llama-3.2-1B-Instruct-quantized.w8a8", {
-            "quantization": "compressed-tensors"
+            "quantization": "compressed_tensors"
```

Comment:
Do we even need these args anymore? It should automatically choose the quantization type

Reply:
I don't think so, I'll remove them instead of changing them